src/routers/user.router.ts

```diff
@@ -0,0 +1,70 @@
+import { Request, Response, Router } from 'express';
+import { createValidator, ValidatedRequest } from 'express-joi-validation';
+
+import { AutoSuggest } from '../models/auto-suggest';
+import { BaseUser, User } from '../models/user';
+import {
+	saveUser,
+	deleteUser,
+	updateUser,
+	createUser,
+	getUserById,
+	getAutoSuggestUsers,
+} from '../controllers/user.controller';
```

Comment:
seems like you confuse `users.service` and `users.controller` coz controller => function/method that process request and they placed here